Algorithm/HEERAK/1027.py

```diff
@@ -0,0 +1,53 @@
+from collections import deque
+from sys import stdin
+input = stdin.readline
+
+def solution(N, arr):
+    # 기울기 구하는 함수
+    slope = lambda A, B: (B[1] - A[1]) / (B[0] - A[0])
+
+    max_cnt = 0
+    for i in range(N):
+        # i번째 건물 좌표
+        A = (i, arr[i])
+
+        # 오른쪽으로 가까운 건물부터 확인
+        cur_slope = None
+        right_cnt = 0
+        for r in range(i + 1, N):
+            # r번째 건물 좌표    
+            B = (r, arr[r])
+            # i번째 건물과 r번째 건물 기울기
+            slope_right = slope(A, B)
+            # 이전 기울기와 현재 기울기를 비교해서 현재기울기가 크면 보이므로
+            if cur_slope is None or cur_slope < slope_right:
+                cur_slope = slope_right
+                right_cnt += 1
+
+        # 왼쪽으로 가까운 건물부터 확인
+        cur_slope = None
+        left_cnt = 0
+        for l in range(i - 1, -1, -1):
+            # l번째 건물 좌표
+            B = (l, arr[l])
+            # i번째 건물과 l번째 건물 기울기
+            slope_left = slope(A, B)
+            # 이전 기울기와 현재 기울기를 비교해서 현재기울기가 작으면 보이므로
+            if cur_slope is None or cur_slope > slope_left:
```

Comment:
```suggestion
            if  cur_slope > slope_left or cur_slope is None:
```
반대로 작성하면 더 좋습니다. 이유는 비밀로 하겠습니다 ㅎㅎ